### PR TITLE
values() error message

### PR DESCRIPTION
**Pull request by wdwatkins**
#304

### CODE REVIEW

File: R/values-webgeom.R [hunk 1]
```diff
@@ -32,9 +32,12 @@ setMethod(f = "values<-",signature(.Object = "webgeom"), definition = function(.
   if(is.na(value[1])){
     .Object@GML_IDs <- as.character(NA)
   } else {
+    gmlID <- fetchGML_IDs(.Object)
```
Comment (wdwatkins) [inline R/values-webgeom.R]: Just test that the null value gets caught?

**Comment by jread-usgs** [comment]
Also, please add your info in the authors section of the DESCRIPTION file

**Comment by jread-usgs** [comment]
Looks like windows is failing w/ the show method for datagroup https://ci.appveyor.com/project/jread-usgs/geoknife/build/0.2.2.395#L758

not sure why...

**Comment by wdwatkins** [comment]
cert error??

**Comment by jread-usgs** [comment]
Looks good

**Pull request merged**